public/js/extra/chart-area.js

```diff
@@ -0,0 +1,146 @@
+// Set new default font family and font color to mimic Bootstrap's default styling
+Chart.defaults.global.defaultFontFamily = "Nunito", "-apple-system,system-ui,BlinkMacSystemFont,'Segoe UI',Roboto,'Helvetica Neue',Arial,sans-serif";
+Chart.defaults.global.defaultFontColor = '#858796';
+
+function numberFormat(number, decimals, decPoint, thousandsSep) {
+  // *     example: number_format(1234.56, 2, ',', ' ');
+  // *     return: '1 234,56'
+  number = (number + '').replace(',', '').replace(' ', '');
+  var n = !isFinite(+number) ? 0 : +number,
+    prec = !isFinite(+decimals) ? 0 : Math.abs(decimals),
+    sep = (typeof thousandsSep === "undefined") ? ',' : thousandsSep,
+    dec = (typeof decPoint === "undefined") ? '.' : decPoint,
+    s = '',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mamoon97150/Blog-PHP/pullRequest?prid=7234961)
```suggestion
    s = "",
```